topics/riscv/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: risc-v, riscv-cpu, riscv-core
+created_by: carlosdelfino
```

Comment:
```suggestion
```

This field is intended to be who created the subject of the topic, not the topic itself. Can we remove this line or add the inventor(s) of Risc-V?

Reply:
I removed the tag as suggested, The inventors of RISC-V is an organization made up of other companies and engineers.